src/functions/template-tags/google-map.php

```diff
@@ -66,20 +68,37 @@ function tribe_get_map_link( $post_id = null ) {
 		$url = 'https://maps.google.com/maps?f=q&amp;source=s_q&amp;hl=en&amp;geocode=&amp;q=' . urlencode( trim( $to_encode ) );
 	}
 
+	/**
+	 * Allows filtering the Google Maps URL for the given event.
+	 * 
+	 * @since 4.6.24
+	 * 
+	 * @param string $url     The URL to Google Maps.
+	 * @param int    $post_id The event post ID.
+	 */
 	$url = apply_filters( 'tribe_events_google_map_link', $url, $post_id );
 	$output = esc_url( $url );
 
+	/**
+	 * Allows filtering the escaped Google Maps URL for the given event.
+	 * 
+	 * @since 4.6.24
+	 * 
+	 * @param string $output The escaped URL to Google Maps.
+	 */
 	return apply_filters( 'tribe_get_map_link', $output );
 }
 
 /**
  * Returns a formed HTML link to Google Maps for the given event.
+ * 
+ * @since 4.6.24
  *
  * @category Events
  *
- * @param string $postId
+ * @param string $postId The event post ID.
```

Comment:
I saw that you had change the `$postId` to `int` above. Does this one also need to be updated?
```suggestion
 * @param int $postId The event post ID.
```